WordPress/openverse-frontend
# Fix homepage RTL

## Fixes


Fixes #765 by @sarayourfriend

## Description


Fixes the homepage RTL problems.

1. The logo and brand mark were in a single SVG making it impossible for them to re-order based on reading direction. Splitting them into two separate SVGs makes this possible. It does make using them a little bit more of a hassle, but this shouldn't be something we're using in a bespoke manner more than a few times, so I don't see this as a maintainability problem. If these logos or brand marks ever change it'll require careful manual positioning fixes anyway regardless of if it's a single or two SVGs.
2. Some non-RTL compliant positioning classes were being used, so this switches them to use the relative margin/padding classes.

## Testing Instructions

Load up this branch and run `pnpm dev`.

1. Visit `/` and verify that the homepage looks good at all screen widths
2. Visit `/ar` and verify that the homepage looks good at all screen widths and that text is aligned
3. Visit `/404` and verify that the logo and brand mark look good
4. Visit `/ar/404` and verify that the logo and brand mark look good

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
